README.md

```
@@ -229,32 +229,32 @@ section in the connector-test-kit-rs README.
 When working on a feature or bugfix spanning adapters and query-compiler code, you will need sibling
-PRs in `prisma/prisma` and `prisma/prisma-engines`. Locally, each time you run
+PRs in `prisma/orm` and `prisma/prisma-engines`. Locally, each time you run
 `DRIVER_ADAPTER=$adapter make test-qe`, tests use the adapters built from your local `../prisma`
 clone.
 
-In CI we need to denote which branch of `prisma/prisma` should be consumed. By default CI clones the
+In CI we need to denote which branch of `prisma/orm` should be consumed. By default CI clones the
 `v7` branch, which will not include your local adapter changes. To test in integration, add the
 following tag to your PR description on a separate line:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)